### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Vasyl-Poremchuk**

### CODE REVIEW

**Comment by Vasyl-Poremchuk** [comment]
image

**Review by KatrinLazarenko — APPROVED** [review]
Looks nice

File: static/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+.sidebar li {
+    font-size: 1.2rem;
+    padding: 0.5em 0;
+    list-style: none;
+    border-bottom: 1px solid lightgray;
+}
+
+.container ul {
+    list-style: square;
+}
```
Comment (Y-Havryliv) [inline static/css/styles.css:17]: add blank lines to the end of the files where you don't have it

File: templates/base.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {% block title %}
```
Comment (Y-Havryliv) [inline templates/base.html:4]: using 2 whitespaces is the best practice
https://github.com/mate-academy/py-task-guideline/blob/main/html_settings/README.MD
please, change everywhere

File: static/css/styles.css (hunk 1 above)
Comment (Vasyl-Poremchuk) [reply static/css/styles.css]: If no blank line has been added to the file, GitHub marks it with a red circle?)

File: templates/base.html (hunk 2 above)
Comment (Vasyl-Poremchuk) [reply templates/base.html]: Done!)

File: static/css/styles.css (hunk 1 above)
Comment (Y-Havryliv) [reply static/css/styles.css]: yep

**Review by Y-Havryliv — APPROVED** [review]
GJ!

**Pull request closed without merging**